lmms_eval/models/qwen_vl.py

```diff
@@ -228,21 +228,25 @@ def _collate(x):
                     until = [until]
                 elif not isinstance(until, list):
                     raise ValueError(f"Expected `gen_kwargs['until']` to be of type Union[str,list] but got {type(until)}")
+
+            if isinstance(contexts, tuple):
+                contexts = list(contexts)
```

Comment:
not sure if this is the best way to handle this, but considering we use item assignment I thought this was ok. Let me know if you have any better ideas.